### PR TITLE
ci yaml file added

### PR DESCRIPTION
**Pull request by JakeFantin**
Added a yaml file for my github actions CI workflow.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "2.7.4"
+ruby "2.7.2"
```
Comment (JacksonStark) [inline Gemfile]: A gem versioning change may be better suited for a separate branch

File: .github/workflows/ruby.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        ports:
+        - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e
```
Comment (JacksonStark) [inline .github/workflows/ruby.yml]: Binding to an immutable SHA-1 may offer more reliability. But it does come with tradeoffs, as  hosted images toolsets (e.g. ubuntu-latest) move forward, and in the event of a breaking issue, actions may react with a patch to a major version to compensate. Meaning not binding to a major version (eg. ruby/setup-ruby@v1) may prevent you from taking advantage of bug fixes, critical functionality, and security fixes

File: .github/workflows/ruby.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+name: Ruby
```
Comment (JacksonStark) [inline .github/workflows/ruby.yml]: CI may be a more suitable name

**Review by rhurtis — COMMENTED** [review]
Looks good to me, just curious as to why you went with node version 14.x though.

**Pull request merged**